app.js

```diff
@@ -1,28 +1,24 @@
+//packages
 var Discord = require("discord.js")
-var bot = new Discord.Client();
+var client = new Discord.Client();
+var ffmpeg = require("ffmpeg-binaries")
+//configuration stuff
+const {apikey} = require("./config");
+const {prefix} = require("./config");
```

Comment:
check in a copy of your api-keys.js (api-keys-template.js) with the keys blanked out for easy usage and recovery on loss of your local

could destructure in a single statement:  const {apikey, prefix} = require("./config");
also, why not just a config object? { apiKey: "x", prefix: "xp" }

Reply:
Checked in.